src/components/Drawer/Drawer.scss

```diff
@@ -0,0 +1,90 @@
+$open-transition-duration: 170ms;
+$icon-size: 26px;
+
+.Drawer {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    bottom: 0px;
+    right: 100px;
+    width: var(--drawer-width);
+    max-height: 100vh;
+    box-shadow:  0 5px 10px rgba(0,0,0,.1);
+    z-index: 100;
+    transform: translateY(100%) translateY(calc(-1 * var(--drawer-head-height)));
+    transition: var(--transition-duration);
+    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+
+    &--isOpen {
+        transform: translateY(0%);
+    }
+
+    &__head {
+        padding: 0 var(--spacing-5x);
+        display: flex;
+        align-items: center;
+        height: var(--drawer-head-height);
+        background-color: var(--drawer-head-background-color);
+        border-radius: 4px 4px 0 0;
+    }
+
+    &__headTitle {
+        font-weight: 200;
+        color: var(--color-neutral-10);
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+
+    &__headButtons {
+        flex-shrink: 0;
+        margin-left: auto;
+
+        button {
```

Comment:
Using nested selectors is not recommended practice in BEM, especially if there is no strong reason for it.
In this case `&__headButtons button` can be selected just by `&__extendButton`